src/v2/Apps/Auction/AuctionApp.tsx

```diff
@@ -29,14 +29,14 @@ export const AuctionApp: React.FC<AuctionAppProps> = ({
 }) => {
   const { contextPageOwnerType, contextPageOwnerSlug } = useAnalyticsContext()
 
+  const showLotStandings = me?.showLotStandingsTab?.length && !sale.isClosed
```

Comment:
We can rename the aliased field below in the query from `showLotStandings` to `showActiveBids` to better reflect the UI.

And we'll also want to update [the test](https://github.com/artsy/force/blob/main/src/v2/Apps/Auction/__tests__/AuctionApp.jest.tsx#L143) to check for this condition.

Reply:
I'd also rename the variable to `showActiveBids` then

Reply:
Confirming that `isClosed` will only be false on `sale` once the last lot is closed (if the sale has cascading end times). Pretty sure yes, but just making sure `isClosed` doesnt check if sale's `end_at` field is in the past because the lots could still be open if that's the case.